sickbeard/scene_exceptions.py

```diff
@@ -320,7 +320,7 @@ def _xem_exceptions_fetcher():
             if not parsedJSON['data']: 
                 logger.log(u"No data returned from XEM when checking scene exceptions. Update failed for {0}".format
                             (sickbeard.indexerApi(indexer).name), logger.DEBUG)
-                continue
+                break
```

Comment:
So why aren't we breaking on:
`if parsedJSON['result'] == 'failure':`
for example?